classes/locations.php

```diff
@@ -795,23 +795,23 @@ function ajax_save_lat_lng() {
 
 			// If we're missing a var, return false.
 			if ( empty( $_POST['sm_lat'] ) || empty( $_POST['sm_lng'] ) || empty( $_POST['sm_id'] ) ) {
-				die( __( "It doesn't look like that worked either. Please try again later.", 'simplemap' ) );
+				die( __( "It doesn't look like that worked either. Please try again later.", 'SimpleMap' ) );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->